platforms/hyperledger-fabric/configuration/roles/create/genesis/tasks/main.yaml

```diff
@@ -8,11 +8,7 @@
 - name: "Create genesis block"
   shell: |
     cd {{ build_path }}
-    {% if '2.' in network.version %}
-    ./configtxgen -profile {{ genesis.name }} -channelID syschannel  -outputBlock ./channel-artifacts/genesis.block
-    {% else %}
-    ./configtxgen -profile {{ genesis.name }} -outputBlock ./channel-artifacts/genesis.block
-    {% endif %}
+    ./configtxgen -profile {{ genesis.name }} -channelID syschannel -outputBlock ./channel-artifacts/genesis.block
```

Comment:
Is this tested for both 1.4.x and 2.x versions?

Reply:
tested for 1.4.4 and it is same for 2.x